obstacle_stop_planner_refine/test/src/test_obstacle_stop_planner.cpp

```diff
@@ -27,6 +27,34 @@ using sensor_msgs::msg::PointCloud2;
 using autoware_planning_msgs::msg::Trajectory;
 using namespace std::chrono_literals;
 
+// FIXME: Rename function name
+Trajectory convertPointsToTrajectoryWithYaw2(
+  const std::vector<Point3d> & points)
+{
+  Trajectory trajectory;
+  for (size_t i = 0; i < points.size(); i++) {
+    autoware_planning_msgs::msg::TrajectoryPoint traj_point;
+    traj_point.pose.position = autoware_utils::toMsg(points[i]);
+    double yaw = 0;
+    if (i > 0) {
+      const double dx = points[i].x() - points[i - 1].x();
+      const double dy = points[i].y() - points[i - 1].y();
+      yaw = std::atan2(dy, dx);
+    } else if (i == 0 && points.size() > 1) {
+      const double dx = points[i + 1].x() - points[i].x();
+      const double dy = points[i + 1].y() - points[i].y();
+      yaw = std::atan2(dy, dx);
+    }
+    const double roll = 0;
+    const double pitch = 0;
+    tf2::Quaternion quaternion;
+    quaternion.setRPY(roll, pitch, yaw);
+    traj_point.pose.orientation = tf2::toMsg(quaternion);
+    trajectory.points.push_back(traj_point);
+  }
+  return trajectory;
+}
```

Comment:
Should define functions like `calcYaw(p1, p2)` and `createQuaternionFromYaw(yaw)`.